<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-VVPR8BFK" ignored="0" name="Test Maven Testing" tests="5" failures="1" timestamp="10 Nov 2018 05:43:46 GMT" time="0.546" errors="0">
  <testcase name="sub" time="0.006" classname="com.qa.tests.DemoTest"/>
  <testcase name="sum" time="0.0" classname="com.qa.tests.DemoTest"/>
  <testcase name="excelDataValidate" time="0.45" classname="com.qa.tests.DemoTest1">
    <failure type="java.lang.AssertionError" message="expected [user2] but found [user1]">
      <![CDATA[java.lang.AssertionError: expected [user2] but found [user1]
at com.qa.tests.DemoTest1.excelDataValidate(DemoTest1.java:32)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- excelDataValidate -->
  <testcase name="sub1" time="0.0" classname="com.qa.tests.DemoTest1"/>
  <testcase name="sum1" time="0.001" classname="com.qa.tests.DemoTest1"/>
</testsuite> <!-- Test Maven Testing -->
